pytorch_lightning/metrics/functional/classification.py

```diff
@@ -568,6 +582,12 @@ def auroc(
         >>> auroc(x, y)
         tensor(0.5000)
     """
+    rank_zero_warn(
+        "This `auroc` was deprecated in v1.2.0 in favor of"
+        " `pytorch_lightning.metrics.functional.auroc import auroc`."
+        " It will be removed in v1.4.0", DeprecationWarning
+    )
+
     if any(target > 1):
```

Comment:
can we internally call the new function so we do not need to maintain the code in 2 places till 1.4?

Reply:
Done :]